Gems/Atom/Feature/Common/Tools/ColorGrading/cmdline/Env_Tools.bat

```diff
@@ -0,0 +1,85 @@
+@echo off
+REM 
+REM Copyright (c) Contributors to the Open 3D Engine Project
+REM 
+REM SPDX-License-Identifier: Apache-2.0 OR MIT
+REM
+REM
+
+:: Sets ocio and oiio tools for O3DE Color Grading
+
+:: Skip initialization if already completed
+IF "%O3DE_ENV_TOOLS_INIT%"=="1" GOTO :END_OF_FILE
+
+:: Store current dir
+%~d0
+cd %~dp0
+PUSHD %~dp0
+
+:: Initialize env
+CALL %~dp0\Env_Core.bat
+CALL %~dp0\Env_Python.bat
+
+::SETLOCAL ENABLEDELAYEDEXPANSION
+
+echo.
+echo _____________________________________________________________________
+echo.
+echo ~    O3DE Color Grading Tools Env ...
+echo _____________________________________________________________________
+echo.
+
+::SET oiio_bin=%O3DE_PROJECT_PATH%\Tools\oiio\build\bin\Release
+::SET oiiotool=%oiio_bin%\oiiotool.exe
+
+:: Libs that ocio uses
+SET vcpkg_bin=%O3DE_DEV%\Tools\ColorGrading\vcpkg\installed\x64-windows\bin
+:: add to path
+SET PATH=%PATH%;%vcpkg_bin%
+
```

Comment:
Why does a vcpkg installed package live here, and not, eg, in a 3p package?  whats going on here?

Reply:
I had to build the OCIO tools myself, and I used vcpkg to do it... so this is piped in here right now so I can carry on with my work.  I don't want to be blocked by a 3P concern that can be done separately.  I can either remove this before I merge these changes, or just update this in the future.  I need to write up a issue for fully adding/building OCIO and all of it's tools, right now we only build the OpenColorIO.dll that is in the engine bin.